### ISSUE TITLE
Collector docs high-level pipeline architecture image needs to be updated

### ISSUE THREAD

**Issue by flands** [open-telemetry/opentelemetry.io#4734]
**What needs to be changed?** The high-level collector pipeline architecture image needs to be updated

**What is the name + path of the page that needs changed?** https://opentelemetry.io/docs/collector/

**Additional context**: The Jaeger exporter has been removed so the example is not valid. The Jaeger receiver may be removed in the future as well given Jaeger supports OTLP ingestion. The diagram also doesn't show connectors which is the newest component type. In the future, the batch processor will move to receiver/exporter helpers so perhaps best to change that now too. Finally, a nit to remove the OTEL COLLECTOR words below the image.

**Comment by svrnm** [comment]
Thanks @flands. 

I agree that connectors should be visualized as well, although it's hard to have them here properly (would we need an arrow back into the receivers then?)

For jaeger: It depends on how one reads the image, are those left and right sources and targets of telemetry or examples of receivers/exporters. I think it is good to have jaeger (and prometheus) listed here but the way things are could be represented better

We (@open-telemetry/docs-approvers) have to track down the source of that image and figure out a way to update it, if needed.

@open-telemetry/collector-approvers PTAL

**Comment by vitorvasc** [comment]
Hi, folks! It's been a while since the last activity here, but I'm revisiting older issues as part of our triaging efforts. 

I believe this is still a valid issue, @open-telemetry/collector-approvers, PTAL!

/cc @tiffany76, in case it fits the ongoing refactor on Collector docs.

**Comment by mx-psi** [comment]
I think this is valid, yes

**Comment by tiffany76** [comment]
Yes, still valid and hopefully being handled as part of #9494. I'll leave this issue open for now to make sure we address all points listed above.
